### PR TITLE
Test: Update render method for StorageTableTests

### PR DESCRIPTION
**Pull request by Jay-Topher**
## Done

- Update render method for StorageTableTests

### QA steps

- run `yarn test src/app/base/components/node/StorageTables/StorageTables.test.tsx`

## Fixes

[MAASENG-1724](https://warthogs.atlassian.net/jira/software/c/projects/MAASENG/issues/MAASENG-1724)

[MAASENG-1724]: https://warthogs.atlassian.net/browse/MAASENG-1724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by petermakowski — CHANGES_REQUESTED** [review]
The linked JIRA task involves migration of StorageTable that still contains enzyme, namely: `src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx`

File: src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx [hunk 1]
```diff
@@ -201,19 +169,16 @@ describe("StorageTable", () => {
...
-    });
-    wrapper.update();
+    const diskSizeInput = screen.getByRole("spinbutton", { name: "Size (GB)" });
+    await userEvent.clear(diskSizeInput);
+    await userEvent.type(diskSizeInput, "21");
+
     expect(
-      wrapper.find("StorageTable .p-form-validation__message").text()
-    ).toBe(`Error: Only 20GB available in ${pool.name}.`);
+      screen.getByText("Only 20GB available in", { exact: false })
```
Comment (petermakowski) [inline src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx:180]: [You seem to have an assertion that includes the pool name in another test](https://github.com/canonical/maas-ui/pull/4922/files#diff-cfb4d974fd840373ba1661b8f590564db0555933324709ce14047f59651c2bb3R214), what's the reason you couldn't use it here?

File: src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export const StorageTable = ({ defaultDisk, hostId }: Props): JSX.Element => {
         {disks.map((disk, i) => {
           const isBootDisk = disk.id === bootDisk;
           return (
-            <Card key={disk.id}>
+            <Card aria-label="disk" key={disk.id}>
```
Comment (petermakowski) [inline src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.tsx:79]: 👌

**Review by petermakowski — APPROVED** [review]

File: src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx (hunk 1 above)
Comment (Jay-Topher) [reply src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx:180]: Adding this assertion should work now, I had it removed when I was testing for partial matches, I'll add ot now

**Pull request merged**